src/GeositeFramework/js/Map.js

```diff
@@ -130,8 +133,15 @@ require(['use!Geosite',
                     }
             }, 300),
             loadEventFired = false;
+        */
```

Comment:
I think it's still calling a `resize` internally but doesn't seem like there's an exposed method similar to what we were using with 3.19. It seems like it would get called on the MapView/SceneView, but in the docs for MapView, at least, `resize` is only listed as an event:

https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html

(For comparison, it's listed as a method in 3.19: https://developers.arcgis.com/javascript/3/jsapi/map-amd.html#resize)